src/config.h

```diff
@@ -127,6 +133,8 @@ typedef struct {
     bool enable_uw_roll;
     bool enable_buffering;
     bool enable_lean_jumping;
+    bool enable_target_change;
+    int32_t target_lock_mode;
```

Comment:
Can we change it to `TARGET_LOCK_MODE`?

Reply:
Sure. I wasn't sure which to use, so I was trying to match the type in `config_map.c` : `.default_value = &(int32_t){TLM_FULL}`

Reply:
I think that can be changed as well.